readme.md

```diff
@@ -72,6 +72,20 @@ Default: `false`
 
 Format the number using the [Binary Prefix](https://en.wikipedia.org/wiki/Binary_prefix) instead of the [SI Prefix](https://en.wikipedia.org/wiki/SI_Prefix). This can be useful for presenting memory amounts. However, this should not be used for presenting file sizes.
 
+##### digits
```

Comment:
The name here is too ambiguous. Digits could mean the non-fractional digits too.

Reply:
Do you prefer `fractionDigits`?